docs/specification.md

```diff
@@ -54,12 +54,13 @@ For the text elements of classes, this language will be the original language (p
 
 ## File Extensions
 
-ABAP file formats define the file types:
+ABAP file formats use the file types:
 * **`.abap`** stores ABAP source code as plain text
 * **`.acds`** stores source code of ABAP Core Data Services (CDS) as plain text
 * **`.json`** stores content of form-based editors or properties of objects, such as `originalLanguage`, `abapLanguageVersion` and others.
 Details are found in [JSON Files in ABAP File Formats](./json.md)
 * **`.properties`** stores text elements as plain text and [translation texts](properties.md)
+* **`.xdp`** stands for XML Data Packaging and is part of Form Objects (SFPF)
```

Comment:
There is one more missing:
- https://github.com/SAP/abap-file-formats/tree/main/file-formats/csnm

Reply:
I believe the list of file types contributes to a certain transparency of the project.